src/i18n.js

```diff
@@ -169,8 +172,10 @@ const messages = {
     'home.netWarning.testnet': '警告！ 请注意这是测试网。',
     'home.netWarning.mainnet': '警告！ 请注意这是主网。',
     'home.netWarning.desc':'在进行交易之前，请确认您的钱包已连接到正确的网络。使用错误的网络可能会导致代币丢失。',
-    'home.eos2evmDesc.p1':'请使用支持 EOS 网络的钱包或中心化交易所，钱包例如 Anchor、Wombat、TokenPocket等，中心化交易所例如 Binance、Coinbase 等。',
-    'home.eos2evmDesc.p2': '如需转账至下方 EOS 合约地址，请在 Memo 中填写目标 EOS EVM 地址以完成 EOS EVM 充值。',
+    'home.eos2evmDesc.p1':'请使用支持 EOS 网络的钱包，如 Anchor、Wombat、TokenPocket 或中心化交易所如 OKX、Coinbase 等。',
+    'home.eos2evmDesc.p2': '如需转账至以下 EOS 合约地址，请在 memo 中填写目标 EOS EVM 地址，以完成向 EOS EVM 的充值。',
+    'home.eos2evmDesc.p3.testnet': '目前官方支持的代币包括 EOS 和 JUNGLE。',
+    'home.eos2evmDesc.p3.mainnet': '目前官方支持的代币包括 EOS和 USDT（EOS）。',
```

Comment:
Inconsistent parentheses characters used. Also there should be no space after `)`.